### PR TITLE
Remove firebase dependency

### PR DESCRIPTION
**Pull request by timothymorse**
## Description
Demoing auth was great in theory but doesn't easily allow others to test the core functionality of the iOS SDK. This removes that dependency and associated logic so that users can use the app without needing Firebase credentials.

### CODE REVIEW

**Review by oshoubber — APPROVED** [review]
Looks good! Small nits: for `<textField>`s we could move the `placeholder` property as the first so its easier to make this a bit more readable from a PR perspective.

**Comment by timothymorse** [comment]
> Looks good! Small nits: for `<textField>`s we could move the `placeholder` property as the first so its easier to make this a bit more readable from a PR perspective.

Thanks! Those files are auto-generated by Storyboard, so unfortunately not much we can do as far as I'm aware. Moving to SwiftUI would for sure fix that, though :)

**Comment by oshoubber** [comment]
> > Looks good! Small nits: for `<textField>`s we could move the `placeholder` property as the first so its easier to make this a bit more readable from a PR perspective.
> 
> Thanks! Those files are auto-generated by Storyboard, so unfortunately not much we can do as far as I'm aware. Moving to SwiftUI would for sure fix that, though :)

Oh dang did not know these were auto generated!

File: PactSafe iOS Demo/Controllers/LoginWithCheckboxViewController.swift [hunk 1]
```diff
@@ -33,9 +32,8 @@ class LoginWithCheckboxViewController: UIViewController {
   private func checkStatus(signerId: String, password: String) {
     ps.signedStatus(for: signerId, groupKey: psGroupKey) { needsAcceptance, contractIds in
       if needsAcceptance {
-        let psAcceptanceVc = PSAcceptanceViewController(self.psGroupKey, signerId: signerId, contractIds: contractIds, customData: nil)
-
         DispatchQueue.main.async {
+          let psAcceptanceVc = PSAcceptanceViewController(self.psGroupKey, signerId: signerId, contractIds: contractIds, customData: nil)
```
Comment (Wilsonl9) [inline PactSafe iOS Demo/Controllers/LoginWithCheckboxViewController.swift:36]: Just curious, since this was moved around, what does this do?

Comment (timothymorse) [reply PactSafe iOS Demo/Controllers/LoginWithCheckboxViewController.swift:36]: Dispatches the UI on the main thread rather than any background threads!

**Review by Wilsonl9 — APPROVED** [review]

**Pull request merged**